### PR TITLE
Add user guide for RawKV CDC

### PR DESCRIPTION
**Pull request by pingyu**
Signed-off-by: pingyu <yuping@pingcap.com>

### What is changed?

Add doc for RawKV CDC.

Most content is migrated from https://github.com/tikv/migration/blob/main/cdc/README.md, but:
1. Add TLS related content.
2. Polish some words.

Preview: https://deploy-preview-328--tranquil-haupia-6f9aae.netlify.app/docs/dev/concepts/explore-tikv-features/cdc/cdc/

### Any related PRs or issues?

No.

### Which version does your change affect?

dev.

### CODE REVIEW

**Comment by pingyu** [comment]
@zeminzhou @haojinming PTAL, thanks~

File: content/docs/dev/concepts/explore-tikv-features/cdc.md [hunk 1]
```diff
@@ -8,14 +8,352 @@ menu:
...
+
+##### Deploy a new TiDB/TiKV cluster including TiKV-CDC
+
+When you deploy a new TiDB/TiKV cluster using TiUP, you can also deploy TiKV-CDC at the same time. You only need to add the `kvcdc_servers` section in the initialization configuration file that TiUP uses to start the TiDB/TiKV cluster. Please refer to the configuration [template](https://github.com/tikv/migration/blob/main/cdc/deployments/tikv-cdc/config-templates/topology.example.yaml).
+
+##### Add TiKV-CDC to an existing TiDB/TiKV cluster
+
+You can also use TiUP to add the TiKV-CDC component to an existing TiDB/TiKV cluster. Take the following procedures:
+
+1. Make sure that the current TiDB/TiKV version >= `6.2.0`.
```
Comment (haojinming) [inline content/docs/dev/concepts/explore-tikv-features/cdc.md:55]: May need to remind that tikv need enable `api v2` to deploy `tikv-cdc`.

File: content/docs/dev/concepts/explore-tikv-features/cdc.md [hunk 2]
```diff
@@ -8,14 +8,352 @@ menu:
...
+* `pd`: A comma-separated list of PD endpoints.
+* `config`: The address of the configuration file that TiKV-CDC uses (optional).
+* `data-dir`: Specifies the directory that TiKV-CDC uses to store temporary files for sorting. It is recommended to ensure that the free disk space for this directory is greater than or equal to 500 GiB (optional).
+* `gc-ttl`: The TTL (Time To Live, in seconds) of the service level `GC safepoint` in PD set by TiKV-CDC (optional). It's the duration that replication tasks can suspend, defaults to 86400, i.e. 24 hours. Note that suspending of replication task will affect the progress of TiKV garbage collection. The longer of `gc-ttl`, the longer a changefeed can be paused, but at the same time more obsolete data will be kept and larger space will be occupied. Vice versa.
+* `log-file`: The path to which logs are output when the TiKV-CDC process is running (optional). If this parameter is not specified, logs are written to the standard output (stdout).
+* `log-level`: The log level when the TiKV-CDC process is running (optional). The default value is "info".
+* `ca`: The path of the CA certificate file in PEM format for TLS connection (optional).
```
Comment (haojinming) [inline content/docs/dev/concepts/explore-tikv-features/cdc.md:77]: Only pem format is acceptable?

**Review by zeminzhou — APPROVED** [review]
LGTM～

File: content/docs/dev/concepts/explore-tikv-features/cdc.md (hunk 1 above)
Comment (pingyu) [reply content/docs/dev/concepts/explore-tikv-features/cdc.md]: Fixed, PTAL~

**Review by haojinming — APPROVED** [review]
LGTM

File: content/docs/dev/concepts/explore-tikv-features/cdc/cdc.md [hunk 3]
```diff
@@ -0,0 +1,393 @@
...
+
+
+[Change Data Capture]: https://en.wikipedia.org/wiki/Change_data_capture
+[TiKV API V2]: ../api-v2
+[v6.2.0]: https://docs.pingcap.com/tidb/v6.2/release-6.2.0
+[TiUP]: https://tiup.io
+[TiKV-BR]: ../backup-restore
+[NFS]: https://en.wikipedia.org/wiki/Network_File_System
+[S3]: https://aws.amazon.com/s3/
+[Backup Raw Data]: ../backup-restore/#backup-raw-data
```
Comment (haojinming) [inline content/docs/dev/concepts/explore-tikv-features/cdc/cdc.md:390]:
```suggestion
[Backup Raw Data]: ../../backup-restore/#backup-raw-data
```

File: content/docs/dev/concepts/explore-tikv-features/cdc/cdc.md [hunk 4]
```diff
@@ -0,0 +1,393 @@
...
+[Restore Raw Data]: ../backup-restore/#restore-raw-data
```
Comment (haojinming) [inline content/docs/dev/concepts/explore-tikv-features/cdc/cdc.md:391]:
```suggestion
[Restore Raw Data]: ../../backup-restore/#restore-raw-data
```

File: content/docs/dev/concepts/explore-tikv-features/cdc/cdc.md [hunk 5]
```diff
@@ -0,0 +1,393 @@
...
+2) Record `backup-ts` from backup result in *Step 1*.
+3) Restore to downstream cluster. Refer to [Restore Raw Data].
+4) Create changefeed with `--start-ts=<backup-ts>`.
+
+
+[Change Data Capture]: https://en.wikipedia.org/wiki/Change_data_capture
+[TiKV API V2]: ../api-v2
+[v6.2.0]: https://docs.pingcap.com/tidb/v6.2/release-6.2.0
+[TiUP]: https://tiup.io
+[TiKV-BR]: ../backup-restore
```
Comment (haojinming) [inline content/docs/dev/concepts/explore-tikv-features/cdc/cdc.md:387]:
```suggestion
[TiKV-BR]: ../../backup-restore
```

**Pull request merged**